app/helpers/formatting_helper.rb

```diff
@@ -217,9 +217,9 @@ def annotate(text)
       text.gsub!(/\s*( *\[)([^\.].*? .*?)(\])/m).each_with_index do |_match, index|
         %(<a href="#annotation-#{ index + 1 }" id="annotation-mark-#{ index + 1 }">#{ index + 1 }</a>)
       end
-      render 'notes/annotated_text', text: text, annotations: annotations.flatten
+      render 'notes/annotated_text.html', text: text, annotations: annotations.flatten
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.